spm_non_sphericity/batch.m

```diff
@@ -90,6 +90,7 @@
 matlabbatch{4}.spm.stats.results.conspec.threshdesc = 'none';
 matlabbatch{4}.spm.stats.results.conspec.thresh = 0.001;
 matlabbatch{4}.spm.stats.results.conspec.extent = 0;
+matlabbatch{4}.spm.stats.results.conspec.conjunction = 1;
```

Comment:
@AlexBowring: looking at the batch file, the "conjunction" option was also amended. Was this intended?